packages/react-app/src/theme.js

```diff
@@ -8,11 +8,13 @@ const theme = extendTheme({
       default: "#088484",
       text: "#026262",
       background: "#C8F5FF",
+      cardBackground: "#E9FBFF",
```

Comment:
**Suggestions**
- Maybe we can use a more generalised name here something like `primary` or `brand` and also add shades to it if necessary as mentioned [here](https://v1.chakra-ui.com/docs/styled-system/theming/customize-theme#customizing-theme-tokens)
- Extending chakra's base components to match our SRE design styles, so there is less repetition and will be easier to maintain as mentioned [here](https://chakra-ui.com/docs/styled-system/customize-theme#customizing-single-components).
    eg : 
    ```JS
    <Button variant="sre-button" />
    ```
- Setting up sensible `baseStyles` for mostly used components like `Text` and all, for eg : Every time we `Text` component we don't need pass `color={primaryFont}` its default color will be our Theme color. 

PS: We can completely ignore all of this or some and can implement it in the future if we want 🙌

Reply:
All of these are great suggestions :) Let's go for it.

I'd say, let's finish this PR as it's... and make another one with the refactor. But, up to you!

Reply:
Yes, I will create an issue to set up a proper theme and would love to work on it, once the #175  is almost done 🙌